onnx/gen_proto.py

```diff
@@ -214,7 +214,7 @@ def main():  # type: () -> None
     parser.add_argument('--protoc_path',
                         default='',
                         help='path to protoc for proto3 file validation')
-    parser.add_argument('stems', nargs='*', default=['onnx', 'onnx-operators'],
+    parser.add_argument('stems', nargs='*', default=['onnx', 'onnx-operators', 'onnx-data'],
```

Comment:
i wasn't aware that the onnx-data was something new added in this PR. Why create a separate proto file?

Reply:
This extension does not affect the IR representation used for ONNX models ... it impacts only the representation of test-data. The main goal is to avoid bumping the IR_VERSION unnecessarily. Keeping the proto files separate is helpful as it clarifies that the data-serialization does not impact the model IR or IR_VERSION. Do you see reasons for keeping them together in one proto (other than the challenges discussed below for importing etc.)?

Reply:
My concern is that the `TensorProto` which is also data-serialization is kept in the original proto. We also see references of it in `initializers` and `attributes`. So the data serialization is impacting the model IR right now (although for sequence and map it is currently not the case, but then we must declare that these types will not supported as initializers or attributes).

Reply:
Discussed offline, consensus is to keep current architecture.